### DISCUSSION TITLE
Grandfathered Free Tier?

### DISCUSSION THREAD

**Discussion by jtkeyva** [supabase/supabase#8259] [Questions]
I have been using the free tier for several months and have 2 organizations with 2 projects each that I have been using to develop on aand learn Supabase with. I like to go back to an older project to reference my data tables and views but now I have to pause all others and restore one just to review something that I've been reviewing for months without issue.

I love Supabase, but this "now you can only have 2 active projects" is frustrating after having been able to have 4 active at the same time. It feels like something has been taken away from us early adopters. Is there any way to grandfather us to the way it was before or allow us 4 active projects?

Thanks

**Comment by wcooperscott** [comment]
As Supabase grows and better evaluates a sustainable business model, it's not that frustrating to see them make reasonable changes. It is all open source. So, if you'd like, you can try and host them on your own AWS account or on your own home server.

😀👍

**Reply by jtkeyva** [reply]
Ya I hear you, I'm grateful for what they offer for free. It's just a real bummer for people who have been using it to build their app for several months and now have to stop and change gears to manually transfer data, files and settings and then setup a home server (which is quite a bit of work compared to something like Strapi).

I'm all in on Supabase but this really puts a damper on progress which is why a grandfather clause (perhaps even just for 3-6-12 months) is a reasonable ask. Taking something away is a bad customer experience and this is a first to me with Supabase.
